pkg/dashboard/router.go

```diff
@@ -69,20 +72,23 @@ func GetRouter(setters ...Option) *mux.Router {
 	router.Handle("/namespaces", NamespaceList(*opts))
 
 	// root
-	router.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		// catch all other paths that weren't matched
-		if r.URL.Path != "/" && r.URL.Path != opts.BasePath && r.URL.Path != opts.BasePath+"/" {
-			klog.Infof("404: %s", r.URL.Path)
-			http.NotFound(w, r)
-			return
-		}
-
-		klog.Infof("redirecting to %v", path.Join(opts.BasePath, "/namespaces"))
-		// default redirect on root path
-		http.Redirect(w, r, path.Join(opts.BasePath, "/namespaces"), http.StatusMovedPermanently)
-	})
+	router.HandleFunc("", rootHandler)
```

Comment:
Is this even necessary since we are adding a `/` to the basePath by default in the cmd/dashboard.go?

If it isn't, then I would prefer keeping the old flow without using a global variable.